R/pkg.R

```diff
@@ -195,13 +195,12 @@ getPackageRecords <- function(pkgNames,
 
   # Prior recursive steps may have already computed this package record and
   # its recursive dependencies. Avoid constructing this package record.
-  priorPkgRecords <- c()
-  for (pkgName in pkgNames) {
-    if (exists(pkgName, envir = .visited.packages)) {
-      append(priorPkgRecords, get(pkgName, envir = .visited.packages))
-    }
+  priorPkgRecords <- dropNull(lapply(pkgNames, function(pkgName) {
+    get0(pkgName, envir = .visited.packages, ifnotfound = NULL)
```

Comment:
```suggestion
    .visited.packages[[pkgName]]
```

as `get0` was introduced with R 3.2.0, and so (in theory) could be too recent (RStudio still declares R 3.0.1 as the minimum required version; not sure what the minimum for RSC is now.)

Reply:
RSC declares R 3.1 as its minimum; I've changed this to use `exists` and `get`.